client/prisma/schema.prisma

```diff
@@ -4,48 +4,60 @@ datasource db {
 }
 
 generator client {
-  provider = "prisma-client-js"
+  provider        = "prisma-client-js"
   previewFeatures = ["extendedIndexes"]
 }
 
 model Voter {
-    id                Int @default(autoincrement()) @id
-    address           String @unique
-    // This is storing a uint256
-    votes             Decimal @db.Decimal(78, 0)
-    firstSeenBlock    Int @map("first_seen_block")
-    updatedAt         DateTime @default(now()) @map("updated_at")
-    createdAt         DateTime @default(now()) @map("created_at")
-
-    @@map("voters")
+  id             Int      @id @default(autoincrement())
+  address        String   @unique
+  // This is storing a uint256
+  votes          Decimal  @db.Decimal(78, 0)
+  firstSeenBlock Int      @map("first_seen_block")
+  updatedAt      DateTime @default(now()) @map("updated_at")
+  createdAt      DateTime @default(now()) @map("created_at")
+
+  @@map("voters")
 }
 
 model Proposal {
-    id          Int @default(autoincrement()) @id
-    // This is storing a uint256
-    proposalId  String @unique @map("proposal_id")
-    description String @default("")
-    createdAt   DateTime @default(now()) @map("created_at")
+  id          Int      @id @default(autoincrement())
+  // This is storing a uint256
+  proposalId  String   @unique @map("proposal_id")
+  description String   @default("")
+  createdAt   DateTime @default(now()) @map("created_at")
 
-    @@map("proposals")
+  @@map("proposals")
 }
 
 model Listener {
-    lastSeenBlock Int @map("last_seen_block") @unique
+  lastSeenBlock Int @unique @map("last_seen_block")
 
-    @@map("listener")
+  @@map("listener")
 }
 
 model Lockup {
-  id        Int @default(autoincrement()) @id
-  lockupId  Int
-  user      String
-  amount    Decimal @db.Decimal(78, 0)
-  end       DateTime
-  points    Decimal @db.Decimal(78, 0)
-  createdAt DateTime @default(now()) @map("created_at")
+  id           Int           @id @default(autoincrement())
+  lockupId     Int
+  user         String
+  amount       Decimal       @db.Decimal(78, 0)
+  end          DateTime
+  points       Decimal       @db.Decimal(78, 0)
+  createdAt    DateTime      @default(now()) @map("created_at")
+  transactions Transaction[]
+  active       Boolean
 
   @@unique([lockupId, user])
   @@index([user], type: BTree)
   @@map("lockups")
 }
+
+model Transaction {
+  hash      String   @unique
+  event     String
+  createdAt DateTime @default(now())
```

Comment:
Suggestion: the use of camel case for column names means quotes are required when writing queries on the sql console. Which is a bit of a pain in the neck :)

Moving forward, in the governance repo can we take the convention to have all column names lower case with underscore? That's what we use in all other projects.

I'm not familiar much with prisma but I think just adding `@map("created_at")` might do the trick?  The column name in the DB will be `created_at` while the model defines the attribute as createdAt

Same comment for Transaction.lockupId below.

Reply:
the pains of sql and case insensitivity :)

Reply:
Yeah, this makes total sense. I think as long as the snake case`@maps()` are included, it's not so bad, but moving forward let's abandon camel case column names when using SQL.